docs/2.8.0/docs/kubernetes/04-deploy-daml-enterprise/01-helm-deployment/02-participant-deployment.rst

```diff
@@ -0,0 +1,119 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Install ``canton-participant`` Chart
+####################################
+
+Objectives
+**********
+
+* Customize Helm chart parameters
+* Install Helm chart
+* Verify the status of the deployment
+
+Steps
+*****
+
+1. Inspect the Helm value files generated by Terraform
+======================================================
+
+.. note::
+   The Terraform scripts were used to parameterize the Helm values. If standalone Helm deployment is done without Terraform the value file has to be customized manually. The value file can be found under ``participant.yaml``. Note that in this guide we are deploying the participant node before deploying the domain node. We may do this because we are deploying both onto the same Kubernetes cluster, and we know the `participant node's identity <https://docs.daml.com/canton/usermanual/identity_management.html#default-initialization>`_ beforehand (e.g., that its name is ``participant1``\ ). For more details, see the `relevant section in the canton-domain Helm chart documentation <https://artifacthub.io/packages/helm/digital-asset/canton-domain#bootstrap>`_. In most cases, the operators of the participant and domain nodes differ, hence, a more involved `on-boarding process <https://docs.daml.com/canton/usermanual/identity_management.html#participant-onboarding>`_ has to be used.
```

Comment:
I copied the participant.yaml file and put it in the docs repo. Need to figure out how to link to it.